### ISSUE TITLE
net::ERR_CONNECTION_RESET when checking for updates

### ISSUE THREAD

**Issue by xehpuk** [bitwarden/clients#3279]
### Steps To Reproduce

1. Go to 'Help'
2. Click on 'Check for Updates...'

### Expected Result

Bitwarden tells me if there are updates available.

### Actual Result

[Window Title]
Error

[Main Instruction]
Update Error

[Content]
```
Error: net::ERR_CONNECTION_RESET
    at SimpleURLLoaderWrapper.<anonymous> (node:electron/js2c/browser_init:101:7169)
    at SimpleURLLoaderWrapper.emit (node:events:526:28)
```

[OK]

### Screenshots or Videos

![grafik](https://user-images.githubusercontent.com/3484906/184033504-2617065c-33d1-4b24-8faa-d9af48493248.png)

### Operating System

Windows

### Operating System Version

10.0.22000.832

### Installation method

Direct Download (from bitwarden.com)

### Build Version

2022.8.0

**Comment by patrickhlauke** [comment]
seems this has been an ongoing electron issue https://github.com/electron-userland/electron-builder/issues/2398

**Comment by djsmith85** [comment]
Closing this now as this was resolved upstream with `electron-updater 6.1.0`
